etl/steps/data/garden/wb/2024-09-09/food_prices_for_nutrition.meta.yml

```diff
@@ -0,0 +1,325 @@
+
+tables:
+  food_prices_for_nutrition:
+    title: Food prices for nutrition
+    variables:
+      cost_of_an_energy_sufficient_diet:
+        title: Cost of an energy sufficient diet
+        unit: international-$ per person per day in 2021 prices
+        short_unit: $
+        description_short: |-
+          Cost of the least expensive starchy staple for energy balance for a representative person at 2330 kcal/day.
+        description_key:
+          - *description_energy_sufficient_diet
+      affordability_of_an_energy_sufficient_diet__ratio_of_cost_to_food_expenditures:
+        title: "Affordability of an energy sufficient diet: ratio of cost to food expenditures"
+        unit: "%"
+        short_unit: "%"
+        description_short: |-
+          Percentage of the cost of an energy sufficient diet to total food expenditure per capita per day from national accounts.
+        description_key:
+          - *description_energy_sufficient_diet
+      percent_of_the_population_who_cannot_afford_sufficient_calories:
+        title: Share of the population who cannot afford sufficient calories
+        unit: "%"
+        short_unit: "%"
+        description_short: |-
+          Percentage of the total population unable to afford an energy sufficient diet. A diet is considered unaffordable when its cost plus expenditures for basic non-food needs exceed income per capita per day.
+        description_key:
+          - *description_energy_sufficient_diet
+          - *description_key_pct_of_population
+      affordability_of_an_energy_sufficient_diet__ratio_of_cost_to_the_food_poverty_line:
+        title: "Affordability of an energy sufficient diet: ratio of cost to the food poverty line"
+        unit: "%"
+        short_unit: "%"
+        description_short: |-
+          Percentage of the cost of an energy sufficient diet to the food poverty lines.
+        description_key:
+          - *description_energy_sufficient_diet
+          - *description_key_poverty_lines
+      people_who_cannot_afford_sufficient_calories:
+        title: Number of people who cannot afford sufficient calories
+        unit: people
+        short_unit: ""
+        description_short: |-
+          Total number of people who cannot afford an energy-sufficient diet.
+        description_key:
+          - *description_energy_sufficient_diet
+          - This indicator is calculated as the percentage of a country's population that is unable to afford an energy sufficient diet, multiplied by the country's population.
+          - *description_key_number_of_population
+      cost_of_a_healthy_diet:
+        title: Cost of a healthy diet
+        unit: current international-$ per person per day
```

Comment:
That's what the World Bank's metadata say: "The cost of a healthy diet indicator is the cost of purchasing the least expensive locally available foods to meet requirements for energy and food-based dietary guidelines (FBDGs), in current PPP$/person/day, for a representative person within energy balance at 2330 kcal/day."
I suppose it would be best to adjust for inflation.

Reply:
In my last commit, I have adjusted this column for inflation.